jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+* not in keeping with the Jenkins philosophy.
+
+The BDFL may be consulted during the approval phase,
+and is the final arbiter of a submission's approvability as a Draft JEP.
+
+[[approval]]
+==== Approval as Draft JEP
+
+Once JEP meets requirements for structure and formatting,
+the editors will approve the submission as a draft JEP
+by following the steps outlined in the
+<<approve-as-draft, editors' "Approve as Draft" section>>.
+When they are done, the Draft JEP will have an official JEP number and
+the sumbmission PR will have been merged to a matching folder and feature branch
+(i.e the
+`link:https://github.com/jenkinsci/jep/tree/jep-1/jep/1[jep/1]` folder in the
+`https://github.com/jenkinsci/jep/tree/jep-1/[jep-1]` branch).
+
+IMPORTANT: "Approval as Draft" is *not* the same as accepting the JEP.
+
+Editors are not the only ones who can approve as submission.
+Non-editor contributors who have "git push" privileges for the
+JEP repository footnoteref:[repo] may also approve submissions.
+When doing so, the developer must handle the tasks
+that would normally be taken care of by the JEP editors
+(see <<editor-responsibilities, JEP Editor Responsibilities & Workflow>>).
+This includes ensuring the initial version meets the expected standards
+for a Draft JEP.
+
+[[draft]]
+==== Refining a Draft JEP
+
+The version of a JEP that is approved as a Draft JEP
+is rarely the same as the final version that is reviewed and hopefully accepted.
+A Draft JEP often requires further refinement and expansion
+before it is suffiently complete and represents the consensus of the community.
+
+Standards Track JEPs consist of two parts, a design document
+and a reference implementation.
+At minimum, prototype implementation should be co-developed with the JEP,
+as ideas that sound good in principle sometimes turn out to be impractical
+when subjected to the test of implementation.
+
+A JEP's author is responsible for collecting community feedback on a JEP
+before submitting it for review.
+Potential changes to a draft JEP may be discussed further on jenkinsci-dev@.
+However, long open-ended discussions are not recommended on mailing lists.
+Strategies to keep the discussion efficient include:
+
+* setting up a series of in-person, or video-conferencing sessions to
+  discuss the JEP with necessary stakeholders.
+* having the JEP author accept private comments in the early design phases
+* setting up a wiki page, etc.
+* committing and reviewing small concrete changes via Pull Requests
+  rather than large sweeping changes
+
+JEP authors should use their discretion here.
+
+The JEP author may also ask JEP editors for further feedback regarding the
+style and consistency of a JEP and its readiness for review by the BDFL.
+
+As updates are necessary, the JEP author and other contributors
+should push commits to their fork of the JEP repository footnoteref:[repo],
+and submit pull requests targeting the JEP's feature branch.
+
+[[review]]
+==== JEP Review
```

Comment:
Whatever we select as a final review stage (BDFL/Governance Meeting/...), there must be SLAs, at least for the first reply and for the decision. Otherwise the proposals may be killed by ignorance as it sometimes happens in the mailing list now

Reply:
UPD: "ignorance" means that it slips in priority, gets missed, etc.

Reply:
Not sure what an "SLA" by volunteer contributors would look like. If nobody replies then it's automatically approved? :smiling_imp:

Reply:
It could be something like that...

BDFL mode:
1) BDFL should provide initial feedback in 2 weeks and make a decision within 1 month
2) If BDFL does not make a decision, the submitter is eligible to escalate JEP to the governance meeting

Governance meeting mode:
1) The JEP owner is responsible to add JEP discussion to the Governance meeting agenda and to propose JEP reviewers. 
2) Governance meeting approves JEP reviewers or selects other ones, who will act as a steering committee
3) Steering committee should perform review of the proposal (in public) and then reports the result at the governance meeting 1 month later (3 options: approve, reject, another cycle required)
4) If the steering committee recommends approve/reject, the Governance meeting votes for the decision.

Reply:
@oleg-nenashev 
Added text regarding Timeliness